Clarify the query cache default behavior. (#4990)
magento/devdocs: guides/v2.3/graphql/develop/create-graphqls-file.md

The `cacheIdentity` value points to the [class]({{page.baseurl}}/graphql/develop/identity-class.html) responsible for retrieving cache tags.

A query without a `cacheIdentity` will not be cached.

To disable caching for queries declared in another module with a `cacheIdentity` class, the `@cache(cacheable: false)` directive can be used.
This `cacheable` argument is intended to disable caching for queries that are defined in another module.

`@cache(cacheable: false)`

Specifying `@cache(cacheable: false)` or `@cache(cacheable: true)` on a query without a `cacheIdentity` class has no effect: the query will not be cached.

> If a query should **no**t be cached, no `@cache` directive at all should be specified. Specifying `@cache(cacheable: false)`  is superfluous when no `cacheIdentity` is present.

Reviewer: ```suggestion
If a query should **not** be cached, do not specify the `@cache` directive. Specifying `@cache(cacheable: false)`  is superfluous when no `cacheIdentity` is present.
```
